Example/FLEXample/SupportingFiles/Info.plist

```diff
@@ -29,23 +29,6 @@
 	<string>NSPhotoLibraryAddUsageDescription</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Save images</string>
-	<key>UIApplicationSceneManifest</key>
```

Comment:
This works only with `SceneDelegate` configured, else it throws error hence deleted SceneDelegate related plist properties.

Reply:
Let's include this as a major change in the MR description and why we had to do so.

Possible to add the error message as well?

Reply:
It was failing because of having this variable `PRODUCT_MODULE_NAME` in Info.plist file, Bazel doesn't support this variable by default hence replaced it with `PRODUCT_NAME`. And we don't have to delete this entire key:value pair, reverted the deletion. 

More info on why bazel deosn't support `PRODUCT_MODULE_NAME` by default - https://github.com/bazelbuild/rules_apple/blob/master/doc/common_info.md#variable-substitution